src/runner/test-run/index.js

```diff
@@ -83,17 +83,31 @@ export default class TestRun extends Session {
         });
     }
 
-    _addError (err) {
+    async _addError (err) {
         if (err.__sourceIndex !== void 0 && err.__sourceIndex !== null) {
             err.relatedSourceCode = this.test.sourceIndex[err.__sourceIndex];
             delete err.__sourceIndex;
         }
 
+        try {
+            err.screenshotPath = await this.screenshotCapturer.captureError(err);
+        }
+        catch (e) {
+            // NOTE: we should just save an error if we can't make a screenshot for some reason
```

Comment:
`// NOTE: swallow error silently if we can't make screenshots for some reason (e.g. we don't have permissions to the write screenshots file)`. Anyway requires @VasilyStrelyaev check